Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -0,0 +1,889 @@
+        # only one argument which is a dict
+        if type(resources[0]) == dict:
+            api_res_resources = resources[0]
+            resources_group_outputs = add_outputs_from_dict(api_res_resources, resources_fields)
+
+            sandbox = api_res_resources.get("sandbox")
+            if api_res_resources and sandbox:
+                api_res_sandbox = sandbox[0]
+                sandbox_group_outputs = add_outputs_from_dict(api_res_sandbox, sandbox_fields)
+        else:
+            # the resources section is a list of strings
+            resources_group_outputs = {"resources": api_res.get("resources")}
+
+    merged_dicts = {**meta_group_outputs, **quota_group_outputs, **resources_group_outputs, **sandbox_group_outputs}
+
+    return merged_dicts
+
+
+def test_module(
+        client: Client,
+) -> Tuple[str, dict, list]:
+    """
+    If a client was made then an accesses token was successfully reached,
+    therefor the username and password are valid and a connection was made
+    additionally, checks if not using all the optional quota
+    :param client: the client object with an access token
+    :return: ok if got a valid accesses token and not all the quota is used at the moment
+    """
+    output = client.check_quota_status()
+
+    error = output.get("errors")
+    if error:
+        return error[0]
+
+    meta = output.get("meta")
+    if meta is not None:
+        quota = meta.get("quota")
+        if quota is not None:
+            total = quota.get("total")
+            used = quota.get("used")
+            if total <= used:
+                raise Exception(f"Quota limitation has been reached: {used}")
+            else:
+                return 'ok', {}, []
+    raise Exception("Quota limitation is unreachable")
+
+
+def upload_file_command(
+        client: Client,
+        file: str,
+        file_name: str,
+        is_confidential: str = "true",
+        comment: str = "",
+        submit_file: str = "no"
+) -> Tuple[str, Dict[str, List[Dict[str, dict]]], List[dict]]:
+    """Upload a file for sandbox analysis.
+    :param client: the client object with an access token
+    :param file: content of the uploaded sample in binary format
+    :param file_name: name of the file
+    :param is_confidential: defines visibility of this file in Falcon MalQuery, either via the API or the Falcon console
+    :param comment: a descriptive comment to identify the file for other users
+    :param submit_file: if "yes" run cs-fx-submit-uploaded-file for the uploaded file
+    :return: Demisto outputs when entry_context and responses are lists
+    """
+    response = client.upload_file(file, file_name, is_confidential, comment)
+
+    resources_fields = ["file_name", "sha256"]
+    filtered_outputs = parse_outputs(response, resources_fields=resources_fields)
+    entry_context = {f'csfalconx.resource(val.id === obj.id)': [filtered_outputs]}
+
+    if submit_file == "no":
```

Comment:
So in here you can just do
```suggestion
    if submit_file:
```
It looks cleaner